sass/components/_popover.scss

```diff
@@ -75,6 +75,7 @@
 
             &:after {
                 border-color: transparent;
+                border-bottom-color: var(--center-channel-bg);
```

Comment:
So, @sudiptog81, this one appears to be a bit of a trick and as a result, we will need to make a bit of a change to where you have placed this line.

Have you ever used the technique where you use thick borders (3 transparent, 1 coloured) on a zero-width/height element to create visual arrows? It's actually a fairly old technique (a may not be so common anymore) so no worries if not!

Basically, you set the entire border to fully transparent and then add a colour to the opposite border from the direction you want the arrow to point and then position it.

What that means for this PR is that you actually need to move your added line out of the the `bottom`/`right`/`top`/`left` shared selector and into one that is only `bottom` like this:

```scss
&.bottom {
    > .arrow::after {
         border-bottom-color: var(--center-channel-bg);
    }
}
```

There is already a solo `&.bottom` selector, so you can just add to that one.

All that said, it looks like the arrows have actually been permanently hidden so this line may not even be necessary. I need to look into that though, so maybe just go ahead and make this change and we can remove later as needed.

Reply:
Sure, I'll go ahead and make the changes.